dafny/lcf/datalog.g4

```diff
@@ -76,10 +122,25 @@ Port
   : 'call' | 'redo' | 'unify' | 'exit' | 'fail'
   ;
 
+Builtin
+  : 'split_string'
+  | 'sub_string'
+  | 'string_lower'
+  | 'string_chars'
+  | 'length'
+  | 'lists:member' | 'member'
+  | 'lists:reverse' | 'reverse'
+  | 'lists:nth1' | 'nth1'
```

Comment:
Do both forms appear? That is, do you see both `lists:member` and `member` in the trace?

Reply:
In the trace, I see "lists:member", but in the .pl file, I see "member".